### PR TITLE
Externally managed content update

### PR DESCRIPTION
**Pull request by hrshdhgd**
Update the externally managed content. This includes all content, including updates to the rare disease subsets.

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -62220,7 +62221,7 @@ xref: ICD9:307.49 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MEDGEN:167920 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MESH:D012893 {source="MONDO:equivalentTo"}
 xref: SCTID:39898005 {source="MONDO:equivalentTo"}
-xref: UMLS:C4042891 {source="https://orcid.org/0000-0002-3458-4839"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:62223]: This line should be maintained based on the [ticket comment](https://github.com/monarch-initiative/mondo/issues/7189#issuecomment-2322144956) for this term. I added a [bug report](https://github.com/monarch-initiative/mondo/issues/8170).

The

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -177066,6 +177068,7 @@ synonym: "encephalotrigeminal syndrome" EXACT [NCIT:C3391]
 synonym: "fourth phacomatosis" RELATED [GARD:0007706]
 synonym: "leptomeningeal angiomatosis" RELATED [GARD:0007706]
 synonym: "meningeal capillary angiomatosis" RELATED [GARD:0007706]
+synonym: "Sturge Weber Syndrome" EXACT [NORD:1741]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:177071]: This is a new synonym though there is another one (the one below) that is the same, but without the capitalization of "syndrome". I think we should address this capitalization issue at a later time. Making a comment just to bring our attention to this.

**Review by sabrinatoro — APPROVED** [review]
It looks good to me.
@twhetzel, please merge this PR. 
I added a comment about a synonym capitalization that I noticed. I don't think this is urgent to address this minor issue, but I want to bring our attention to it as it might be relevant to the synonym synch project.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:177071]: Thanks for the note. I added a [ticket](https://github.com/monarch-initiative/mondo/issues/8171) about this. 

@sabrinatoro  Can I fix this now? Also, is there a way to contact NORD about the preferred letter case for the synonym, e.g. Sturge Weber syndrome?

Comment (twhetzel) [reply src/ontology/mondo-edit.obo]: I removed the duplicate.

**Pull request merged**